docs/data-masking-function-list.md

```diff
@@ -0,0 +1,871 @@
+
+### Returns
+
+Returns a string with the selected characters masked by a specified `mask_char` or that parameter's default value in the same character set as `str`.
+
+An error occurs if the `str` parameter is not the correct length. 
+
+Returns a NULL value if the `str` is in an incorrect format.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_uk_nin ('CT 26 46 83 D');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------+
+    | mask_uk_nin('CT 26 46 83 D')       |
+    +------------------------------------+
+    | CT ** ** ** *                      |
+    +------------------------------------+
+    ```
+
+## mask_uuid(str [,mask_char])
+
+
+
+Masks a Universally Unique Identifier (UUID).
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| str | No | The string to be masked | String |
+| mask_char | Yes | The masking character | String |
+
+The `str` format is `********-****-****-****-************`.
+
+If the `mask_char` is not specified, the default is '*'. The `mask_char` value can be a multibyte character in any character set and may not be same character set as `str`.
+
+### Returns
+
+A string with the characters masked by a specified `mask_char` or that parameter's default value in the same character set as `str`.
+
+Returns an error if the length of `str` is incorrect.
+
+Returns `NULL` if `str`  is an incorrect length.
+
+### Example
+
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT mask_uuid('9a3b642c-06c6-11ee-be56-0242ac120002');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +-------------------------------------------------------+
+    | mask_uuid('9a3b642c-06c6-11ee-be56-0242ac120002')     |
+    +-------------------------------------------------------+
+    |********_****_****_****_************                   |
+    +-------------------------------------------------------+
+    ```
+
+## masking_dictionary_remove(dictionary_name)
+
+
+
+Removes all of the terms and the dictionary. 
+
+Requires the `MASKING_DICTIONARIES_ADMIN` privilege.
+
+### Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| dictionary_name | No | The dictionary to be removed | String |
+
+
+### Returns
+
+Returns a string value of `1` (one) if the operation is successful or `NULL` if the operation could not find the `dictionary_name`.
+
+### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT masking_dictionary_remove('trees');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------+
+    | masking_dictionary_remove('trees')       |
+    +------------------------------------------+
+    |                                        1 |
+    +------------------------------------------+
+    ```
+
+## masking_dictionary_term_add(dictionary_name, term_name)
```

Comment:
Hi Patrick! We need to add the following note for masking_dictionary_term_add:
'Note that masking_dictionary_term_add uses INSERT IGNORE to add data to mysql.masking_dictionaries table. Some of the points to consider as a result: 
1) if inserted data is greater than mysql.masking_dictionaries.Dictionary or mysql.masking_dictionaries.Term maximum length, the inserted data is truncated without warnings. 
2) Characters not supported by mysql.masking_dictionaries.Dictionary and/or mysql.masking_dictionaries.Term character sets are treated as '?'.
Feel free to change wording :)

Reply:
`data is greater ` -> `data is longer`